### PR TITLE
chore: use new release for the expire and cleanup feature

### PR DESCRIPTION
**Pull request by sqin2019**
expecting workflow runs:
Screenshot 2023-09-08 at 12 14 09 PM

### CODE REVIEW

File: .github/workflows/expire.yml [hunk 1]
```diff
@@ -22,14 +22,33 @@ on:
...
+      matrix:
+        pull: '${{ fromJSON(needs.expire.outputs.pulls) }}'
+    permissions:
+      pull-requests: 'write'
+      contents: 'write'
+      id-token: 'write'
+    uses: 'sqin2019/access-on-demand-fork/.github/workflows/cleanup.yml@main'
+    with:
+      workload_identity_provider: 'projects/901968413216/locations/global/workloadIdentityPools/github-pool-c12c/providers/github-provider'
+      service_account: 'hello-world-c12c-ci-sa@suhongq-dev-041665.iam.gserviceaccount.com'
```
Comment (yolocs) [inline .github/workflows/expire.yml]: These values don't look right...

**Review by yolocs — CHANGES_REQUESTED** [review]

File: .github/workflows/expire.yml (hunk 1 above)
Comment (sqin2019) [reply .github/workflows/expire.yml]: sorry...sloppy copy paste..

Comment (sqin2019) [reply .github/workflows/expire.yml]: updated!

File: .github/workflows/expire.yml [hunk 2]
```diff
@@ -22,14 +22,37 @@ on:
...
+    needs: 'expire'
+    if: '${{ needs.expire.outputs.pulls != ''[]''}}'
+    strategy:
+      matrix:
+        pull: '${{ fromJSON(needs.expire.outputs.pulls) }}'
+    permissions:
+      pull-requests: 'write'
+      contents: 'write'
+      id-token: 'write'
+    uses: 'abcxyz/access-on-demand/.github/workflows/cleanup.yml@main'
```
Comment (yolocs) [inline .github/workflows/expire.yml]: Ping version?
```suggestion
    uses: 'abcxyz/access-on-demand/.github/workflows/cleanup.yml@v0.1.1'
```

Comment (sqin2019) [reply .github/workflows/expire.yml]: done.

**Review by yolocs — APPROVED** [review]

**Pull request merged**